cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestLocationController.java

```diff
@@ -24,6 +24,7 @@
 
 package mil.army.usace.hec.cwms.radar.client.controllers;
 
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
```

Comment:
Is the purpose of this just to have one assertion in each test?

Reply:
pretty much. This lets junit report a failure vs an error and tells other devs which method is being tested. The round trip testing is implemented at the higher integration test layer. Seemed weird to me to test round tripping with a mock server.